.github/workflows/build.yml

```diff
@@ -78,12 +84,25 @@ jobs:
           fi
           echo "CACHE_TO=$CACHE_TO" >> $GITHUB_ENV
 
+      - name: "Generate tags"
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.SERVER_IMAGE }}
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=sha,enable=true,priority=100,prefix=,suffix=,format=short     
+            type=sha,enable=true,priority=100,prefix=${{ github.ref_name }}.,suffix=,format=short     
```

Comment:
@joerunde -- I replicated what you did in https://github.com/IBM/text-generation-router/blob/main/scripts/get_image_tags.sh

Reply:
nice! I'll have to go backport